bitsail-components/bitsail-component-metrics/bitsail-component-metrics-prometheus/src/main/java/com/bytedance/bitsail/component/metrics/prometheus/option/PrometheusPushGatewayOptions.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Bytedance Ltd. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.component.metrics.prometheus.option;
+
+import com.bytedance.bitsail.common.option.CommonOptions;
+import com.bytedance.bitsail.common.option.ConfigOption;
+import com.bytedance.bitsail.common.option.ConfigOptions;
+import com.bytedance.bitsail.common.option.WriterOptions;
+
+public interface PrometheusPushGatewayOptions extends WriterOptions.BaseWriterOptions {
+  String PUSHGATEWAY_PREFIX = CommonOptions.COMMON_PREFIX + "pushgateway-";
+
+  ConfigOption<String> PUSHGATEWAY_HOST_NAME =
+      ConfigOptions.key(PUSHGATEWAY_PREFIX + "host")
```

Comment:
`job.common.metric.prometheus_host` sounds good?

Reply:
I change name like `job.common.metric-prometheus-port` and `job.common.metric-pushgateway-jobname` to make same name structure with old options.

Reply:
we use `_` instead of `-` in other config, so please be consistent here

Reply:
Fixed it, Now it comes the format like `job.common.metric.prometheus_host`.